kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -333,6 +336,10 @@ define([
                             hide: Config.get('tooltip').hideDelay
                         }
                     });
+                    $('td:eq(1)', nRow).find('span.kb-data-staging-folder').off('click').on('click', e => {
+                        $(e.currentTarget).off('click');
```

Comment:
Also in here - if you double click the folder icon, this event will fire multiple times, leading to paths looking like `/folder/folder` instead of `/folder`. Disabling the click event as soon as you click it can prevent that. And since that'll cause a re-render, any other events will get set up correctly at that point.